compiler/util/llvmDebug.cpp

```diff
@@ -18,6 +18,7 @@
  */
 
 #include "llvmDebug.h"
+#include "llvmVer.h"
```

Comment:
We've been sorting includes for these LLVM files into 4 parts:
 1. Include the header for the file we're working on (e.g. llvmDebug.h here)
 2. Include, in alphabetical order, other Chapel compiler headers
 3. Include system headers
 4. Include LLVM headers, in HAVE_LLVM

Here, I think the llvmVer header should be sorted within the other Chapel headers or possibly included within the HAVE_LLVM.